client/src/components/Common/Heading.vue

```diff
@@ -33,18 +49,29 @@ const element = computed(() => {
 <template>
     <div v-if="props.separator" class="separator heading">
         <div class="stripe"></div>
-        <component :is="element" :class="[sizeClass, props.bold ? 'font-weight-bold' : '']">
+        <component
+            :is="element"
+            :class="[sizeClass, props.bold ? 'font-weight-bold' : '', collapsible ? 'collapsible' : '']"
+            @click="$emit('click')">
```

Comment:
These elements are non-interactive, making click-handlers on them inaccessible. I'd prefer if we could use a button here